apps/docs/src/components/home/ThemeSwitcher.tsx

```diff
@@ -0,0 +1,48 @@
+import { useCallback } from "react";
+import { HvOption, HvSelect } from "@hitachivantara/uikit-react-core";
+
+import { useDocsThemeContext } from "../../contexts/DocsThemeContext";
+
+const themesModes = [
+  "pentahoPlus-dawn",
+  "pentahoPlus-wicked",
+  "ds5-dawn",
+  "ds5-wicked",
+  "ds3-dawn",
+  "ds3-wicked",
+];
```

Comment:
what's I was suggesting above is something like this
```suggestion
const themes = ["pentahoPlus","ds5","ds3"];
```